organizers/models.py

```diff
@@ -5,24 +5,16 @@
 from django.db.models import Avg, F
 from django.utils import timezone
 
+from app.settings import MAX_VOTES
```

Comment:
probs better to do this.

```suggestion
from django.conf import settings
MAX_VOTES = getattr(settings, 'MAX_VOTES', 10)
```